ibis/backends/base/sql/alchemy/geospatial.py

```diff
@@ -1,10 +1,8 @@
 from __future__ import annotations
 
-try:
-    import geoalchemy2
-    import geoalchemy2.shape  # noqa: F401
```

Comment:
I was able to remove this because it's always available in the versions of `geoalchemy2` + `shapely` that we support.